### PR TITLE
Update default value for mobile column

### PR DESCRIPTION
**Pull request by martinamarien**
**PR Summary:** 
Updates the default value of `Mobile Layout > Number of columns on mobile` to 1. 

**Why are these changes introduced?**
In the mobile grid update, we set the default mobile column value to 2 for Multicolumn and Collection list. This will create a visual change for merchants who update to the next version of Dawn. To prevent the visual change, we will set the default value to `"1"`, which was the default value for these two sections. 

**What approach did you take?**
Set default to 1

```
"default": "1",
```

**Other considerations**

**Testing steps/scenarios**
- [ ] Test Mobile experience for Multicolumn - should default to 1.
- [ ] Test Mobile experience for Collection list section - should default to 1.

**Demo links**
_Please include a link to a demo store that includes preconfigured sections and settings to allow reviewers to easily test the features you are working on._

- [Editor](https://os2-demo.myshopify.com/?_ab=0&_fd=0&_sc=1&key=426fb1f420e906418fa2448926c74e347c9832a999315511ee1be9880525803a&preview_theme_id=127647350806)

**Checklist**
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: sections/collection-list.liquid [hunk 1]
```diff
@@ -193,7 +191,9 @@
           "value": "2",
           "label": "t:sections.collection-list.settings.columns_mobile.options__2.label"
         }
-      ]
+      ],
+      "default": "1",
+      "label": "t:sections.collection-list.settings.columns_mobile.label",
```
Comment (martinamarien) [inline sections/collection-list.liquid:196]:
```suggestion
      "label": "t:sections.collection-list.settings.columns_mobile.label"
```

File: sections/collection-list.liquid [hunk 2]
```diff
@@ -256,4 +256,4 @@
     }
   ]
 }
-{% endschema %}
\ No newline at end of file
+{% endschema %}
```
Comment (kmeleta) [inline sections/collection-list.liquid:259]: Missing the trailing newline

**Review by kmeleta — APPROVED** [review]
That's better, lgtm

File: sections/collection-list.liquid (hunk 2 above)
Comment (martinamarien) [inline sections/collection-list.liquid:259]:
```suggestion
{% endschema %}

```

**Pull request merged**